themes/edx.org/lms/templates/course_modes/choose.html

```diff
@@ -45,16 +45,11 @@
             });
 
             % if use_ecommerce_payment_flow:
-            $('input[name=verified_mode]').click(function(e){
+            $('button[name=verified_mode]').click(function(e){
                 e.preventDefault();
                 window.location.href = '${ecommerce_payment_page | n, js_escaped_string}?sku=' +
                 encodeURIComponent('${sku | n, js_escaped_string}');
             });
-            $('.v2 button[name=verified_mode]').click(function(e){
```

Comment:
Just want to double check that the V1/V2 stuff we're removing isn't needed anymore (and isn't being used by any running or upcoming experiments)

Reply:
we ran 4 experiments changing the format of track selection a couple years ago that were using the v2 html. they just didn't get cleaned up, so cleaning them up now